README.md

```diff
@@ -15,7 +19,7 @@
 | ![cross][checkmark] Dice-based Combat             | ![check][checkmark] Save/Load System                  | ![cross][cross] Cinematic Cutscenes |
 | ![cross][cross] New Maps/Levels                   | ![cross][cross] New Enemy Types                       | ![cross][cross] Online Multiplayer  |
 | ![check][checkmark] Character Creation            | ![cross][checkmark] Customisable Outfit Colour        | ![cross][cross] Party System        |
-| ![cross][cross] Additional Items in Shop          | ![cross][cross] Split Screen Local Multiplayer        |                                     |
+| ![cross][checkmark] Additional Items in Shop      | ![cross][cross] Split Screen Local Multiplayer        |                                     |
```

Comment:
Although it won't change how it looks, would mine changing it to `![check][checkmark]`? Also change it on the others if you do.